src/terraform/internal-dss/create_grafana_user.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# Create a PostgreSQL data source
+curl "http://admin:$ADMIN_PASSWORD@$GRAFANA_HOST:$GRAFANA_PORT/api/datasources" \
+    -H 'Accept: application/json' \
+    -H 'Content-Type: application/json' \
+    --data "{\"name\":\"DSS Database\",\"type\":\"postgres\",\"url\":\"$DATABASE_HOST:$DATABASE_PORT\",\"access\":\"proxy\",\"user\":\"$DATABASE_USER\",\"password\":\"$DATABASE_PASSWORD\",\"jsonData\": {\"database\":\"$DATABASE_DB\",\"sslmode\":\"disable\",\"postgresVersion\":1500}}"
```

Comment:
Does this correspond to postgres v14? That is what GCP is using

Reply:
Did you test this out? It is quite different from what ended up being required on GCP:

https://github.com/econia-labs/econia/blob/af85fe5131ceea4ac879637c157f682621eabd66/src/terraform/dss-ci-cd/dss/modules/grafana/main.tf#L66-L94

Reply:
> The version does not seem to be important

Not now, at least, but if a PostgreSQL version major gets released then perhaps the new defaults could introduce unexpected behaviors.

Our postgres DSS db Dockerfile pins to v14, as does our GCP deployment for both DSS db and Grafana state, so I'm thinking we should probably create a new Postgres docker compose container for v14 to store Grafana state: this will make it easier to migrate grafana state back and forth between cloud and local, since Grafana defaults to SQLite for storing state